tests/unit_tests/c_wrappers/res/job_queue/test_job_queue_manager_torque.py

```diff
@@ -135,6 +135,9 @@ def create_qstat_f_output(
 """
 )
 
+# A qstat command that lies about its return code on first invocation, then succeeds:
```

Comment:
perhaps make it clear that this variant of qstat not only lies about its return code upon first invocation, but also has no output / produces no file (at least if i understood this correctly) - 

```suggestion
# A qstat command that has no output but rc 0 on first invocation, and then succeeds:
```